### PR TITLE
adds up-down scroll container and interaction

### PR DESCRIPTION
**Pull request by sepowitz**

### CODE REVIEW

File: app/components/scroll-to-arrow.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    }
+  }
+});
+
+export default Ember.Component.extend({
+  styles,
+  sanctu: service(),
+
+  click(e) {
+    let direction = $(e.target).attr("class");
```
Comment (hhff) [inline app/components/scroll-to-arrow.js]: IMO - direction should be a prop on the component ie:

```
sanctu: service(),
direction: 'up',
```

```
{{scroll-to-arrow direction='down'}}
```

Comment (hhff) [reply app/components/scroll-to-arrow.js]: oh `down: false` is fine too~!

Comment (hhff) [reply app/components/scroll-to-arrow.js]: in that case i'd just do 
```
let direction = get(this, 'down') ? 'down', 'up';
```

File: app/components/scroll-to-arrow.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+export default Ember.Component.extend({
+  classNameBindings: [`shouldShow:${styles.show}:${styles.hide}`],
+  styles,
+  sanctu: service(),
+
+  didInsertElement() {
+    this._super(...arguments);
+    this.checkScrollHeight();
+    this.$(window).resize(() => this.checkScrollHeight());
+  },
```
Comment (hhff) [inline app/components/scroll-to-arrow.js]: in `willDestroyElement` you'll want to tear this listener down

Comment (sepowitz) [reply app/components/scroll-to-arrow.js]: k

**Comment by hhff** [comment]
```
didInsertElement() {
  const resize = () => this.checkScrollHeight();
  set(this, 'resizeListener', resize);
  this.$(window).on('resize', resize);
},

willDestroyElement() {
  this.$(window).off('resize', get(this, 'resizeListener'));
}
```

**Pull request merged**